src/main/java/com/enderio/base/common/block/IndustrialInsulationBlock.java

```diff
@@ -8,12 +8,10 @@
 import net.minecraft.world.level.block.*;
 import net.minecraft.world.level.block.entity.BlockEntity;
 import net.minecraft.world.level.block.state.BlockState;
-import net.minecraft.world.level.material.FluidState;
-import net.minecraft.world.level.material.Material;
-import net.minecraftforge.common.Tags;
 
 import java.util.Queue;
 
+// TODO: 1.20.1, why does this do custom work?
```

Comment:
what do you mean by this?

Reply:
I was wondering why this has logic in addition to the base sponge block

Reply:
it voids all fluids, not just water

Reply:
ahh so it does! Thats all good then ill drop the comment and merge :)